source/blog/_posts/2017-10-23-meetnext-recap.md

```diff
@@ -0,0 +1,137 @@
+---
+title: MeetNext - a recap
+tags:
+- community
+- next
+- meetnext
+- shopware
+
+categories:
+- dev
+
+authors: [nd]
+github_link: blog/_posts/2017-10-23-meetnext-recap.md
+
+---
+
+# Introduction
+
+[![Army of Codys](/blog/img/2017-10-23-meetnext-recap/cody_army_thumb.jpg)](/blog/img/2017-10-23-meetnext-recap/cody_army.jpg)
+
+## When
+
+&#35;MeetNext took place from October 18th to 20th
+
+## Where
+
+[![Germania Campus](/blog/img/2017-10-23-meetnext-recap/campus_view_thumb.jpg)](/blog/img/2017-10-23-meetnext-recap/campus_view.jpg)
+
+Münster, Germany
+
+## What
+
+The participants were informed about the current state of the shopware research.
+
+So, what is the `Next` in &#35;MeetNext about? In a nutshell, it is the next big thing. The next big release of shopware, although it will be a different product and 5.x (and possibly 6.x) will be developed simultaneously. They will both receive patches, new features. Which leads us to the next question:
```

Comment:
"Shopware" should be written capitalized when referring to the product Shopware. 
Sorry, @tiagojsag drilled us all on that on 🙈

Reply:
```diff
- They will both receive patches, new features.
+ They will both receive patches and new features.
```

Reply:
![image](https://user-images.githubusercontent.com/973964/32008445-81d0cf84-b9ac-11e7-9c5b-2fb7aac37f10.png)